go-controller/pkg/cni/cni.go

```diff
@@ -199,3 +178,24 @@ func (pr *PodRequest) getCNIResult(podInterfaceInfo *PodInterfaceInfo) (*current
 		IPs:        ips,
 	}, nil
 }
+
+// getAnnotationsOnPod obtains the pod annotation from the cache
+func getAnnotationsOnPod(podLister corev1listers.PodLister, namespace, name string) (map[string]string, error) {
+	var annotations map[string]string
+
+	if err := wait.PollImmediate(1*time.Second, 5*time.Second, func() (bool, error) {
```

Comment:
@trozet is 5 seconds too low?

Reply:
@aojea yes, why change the total timeout from what ExponentialBackoff had? It's about 25 - 30 seconds currently.

Reply:
because based on your comment I thought that long waits make the situation worse, and maybe is preferably to fail fast